### DISCUSSION TITLE
Why is raw value ok but not the final value?

### DISCUSSION THREAD

**Discussion by Pewibe** [jomjol/AI-on-the-edge-device#3044] [Q&A]
Why is the raw value equal to photo condition, but the final value is increased by one? Maybe the digital counter of my water meter is mechanically not optimal but the raw recognition is correct. What kicks the postprocessing to this wrong result? 

![Clipboard01](https://github.com/jomjol/AI-on-the-edge-device/assets/42893315/e0e1ca25-d486-495c-937e-3722b2599bc7)

**Comment by Pewibe** [comment]
Because nobody could help me since daysI like to ask @SybexX if my problem is identical to #2742 and #3025 ?

**Reply by SybexX** [reply]
What value did you set for analogDigitalTransitionStart?
Have you tried increased or reduced the value for "analogDigitalTransitionStart"? 
How does the last digit, does it run normally or does it jump?
Have you tried a different model?

**Comment by Pewibe** [comment]
Sorry for being late with my answers:
1. analogDigitalTransitionStart = 9.2
2. No, have not changed the 9.2
3. Last digit does not jump, it moves like a analog digit.
4. No, I used the default models dig-cont_0620_s3_q.tflite and ana-cont_1208_s2_q.tflite which are not explained in documentation. 

@SybexX, I hope now you are able to answer my original question (raw recognition works but postprocessing turn it into error). As I do not deeply know the software and concepts behind, I believe that models are needed to build the raw value but have nothing to do with postprocessing the raw to the final value (differs by 1,0 as I wrote). What can turn e.g. 314,XXXX to 315,XXXX ?
